react/features/base/tracks/middleware.js

```diff
@@ -84,14 +85,14 @@ MiddlewareRegistry.register(store => next => action => {
 
         // The TRACK_ADDED action is dispatched when a presenter starts a screenshare. Do not create a local fake
         // screenshare participant when multiple stream is not enabled.
-        const skipCreateFakeScreenShareParticipant = local && !getMultipleVideoSupportFeatureFlag(state);
+        const skipCreateFakeScreenShareParticipant = local && !getMultipleVideoSendingSupportFeatureFlag(state);
```

Comment:
Why are we skipping the local SS tile creation for plan-b clients ? If we show the fake SS tile on remote endpoints, shouldn't we create one on the local endpoint as well ?

Reply:
I imagine it's because in planB there's just one video and it can be displayed in the local thumbnail like it used to be? (haven't read the entire PR yet, but I think we should go with whichever path is less complicated)

Reply:
No, I think we decided to show the fake SS tile even if the sender (plan-b) sends a single track. So we are creating a fake tile on the receiver always (plan-b or unified) but not on the local ep if its running plan-b.

Reply:
Oh I think fake SS tile creation gets suppressed here but gets executed when `JitsiTrackEvents.TRACK_VIDEOTYPE_CHANGED` event is fired on the local track when in plan-b. It does get a little confusing if track creation is not handled only in tracks middleware.

Reply:
> When does `TRACK_VIDEOTYPE_CHANGED` gets fired for local track? I'll have to retest this, I thought `TRACK_ADDED` is trigger for plan-b senders instead of `TRACK_VIDEOTYPE_CHANGED` when they start s SS.

my bad, `TRACK_VIDEOTYPE_CHANGED` is not fired for the local track.

Reply:
> To confirm, if we want to also have a plan-b sender see two tiles, I can remove the logic here. However, we will need to ensure the original tile shows an avatar or users will see duplicate tiles. The remaining work is to update make a small modification to computeDisplayModeFromInput for web but I don't know what needs to be done for RN
> 
> >

I think we should go ahead and do this. Whenever multi-stream flag is enabled on a deployment, we would want the behavior to be consistent irrespective of the SDP mode that the endpoints are running in. That means a fake SS tile needs to be added both on the senders and the receivers. Do you agree ?

Reply:
yes at this point we might as well make it consistent through out. Just pushed changes to update  the sender side